src/main/java/gov/usds/case_issues/controllers/KPIController.java

```diff
@@ -0,0 +1,35 @@
+package gov.usds.case_issues.controllers;
+
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import gov.usds.case_issues.services.KPIService;
+
+
+/**
+ * Controller to see information about the current user.
+ */
+@RestController
+@Profile("auth-testing")
```

Comment:
This profile restriction is probably not what you wanted, unless you wanted to have this randomly not work in production. 😁